modules/auxiliary/admin/networking/cisco_dcnm_auth_bypass.rb

```diff
@@ -0,0 +1,156 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'securerandom'
+require 'base64'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Cisco DCNM auth bypass',
+        'Description' => %q{
+          This exploit is able to add an admin account to a Cisco DCNM with credentials you can choose.
+          After that, you can login to the web interface with those credentials.
+          The only necessary condition is the more or less recent connection of an admin as this exploit
+          uses a kind of session stealing.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'MR_ME', # Amazing POC on www.exploit-db.com
+          'Yann Castel (yann.castel[at]orange.com)' # Metasploit module
+        ],
+        'References' =>
+          [
+            [ 'CVE', '2019-15975']
```

Comment:
The author line on L26 implies there's a PoC on exploit-db, can you please add that to the references?

Reply:
Fixed !